linux/requirements.bsh

```diff
@@ -305,7 +268,7 @@ function version_gt()
 #
 # Splits a version string up into its elements and stores them in an array.
 #
-# The last element is always the pre-release/metadata. If there is no pre-release or metadata, the last element will be an empty string.
+# Any non-numerical values are stored in the last element is as pre-release/metadata (including leading/trailing whitespace)
```

Comment:
last element as pre-release

Reply:
Is this true here, or only in `split_version_string_and_remainder`

Reply:
nm. i gather that it is true here as well

Reply:
Yeah, the two functions are fairly similar, but it was useful to have two different behaviors to cover all the different scenarios properly, without adding extra complications.